modules/loop_search_node/loop_search_node.pages_default.inc

```diff
@@ -0,0 +1,107 @@
+  $page = new stdClass();
+  $page->disabled = FALSE; /* Edit this to true to make a default page disabled initially */
+  $page->api_version = 1;
+  $page->name = 'search_node';
+  $page->task = 'page';
+  $page->admin_title = 'search node';
+  $page->admin_description = '';
+  $page->path = 'search';
+  $page->access = array(
+    'plugins' => array(
+      0 => array(
+        'name' => 'role',
+        'settings' => array(
+          'rids' => array(
+            0 => 2,
+          ),
+        ),
+        'context' => 'logged-in-user',
+        'not' => FALSE,
+      ),
+    ),
+    'logic' => 'and',
+  );
+  $page->menu = array();
+  $page->arguments = array();
+  $page->conf = array(
+    'admin_paths' => FALSE,
+  );
+  $page->default_handlers = array();
+  $handler = new stdClass();
+  $handler->disabled = FALSE; /* Edit this to true to make a default handler disabled initially */
+  $handler->api_version = 1;
+  $handler->name = 'page_search_node__panel';
+  $handler->task = 'page';
+  $handler->subtask = 'search_node';
+  $handler->handler = 'panel_context';
+  $handler->weight = 0;
+  $handler->conf = array(
+    'title' => 'Panel',
+    'no_blocks' => 0,
+    'pipeline' => 'standard',
+    'body_classes_to_remove' => '',
+    'body_classes_to_add' => '',
+    'css_id' => '',
+    'css' => '',
+    'contexts' => array(),
+    'relationships' => array(),
+    'name' => 'panel',
+  );
+  $display = new panels_display();
+  $display->layout = 'full-width';
+  $display->layout_settings = array();
+  $display->panel_settings = array(
+    'style_settings' => array(
+      'default' => NULL,
+      'alpha' => NULL,
+      'beta' => NULL,
+      'gamma' => NULL,
+    ),
+  );
+  $display->cache = array();
+  $display->title = 'Search';
+  $display->uuid = '8e14edd0-235e-4fed-862c-469e3caac2de';
+  $display->content = array();
+  $display->panels = array();
+    $pane = new stdClass();
+    $pane->pid = 'new-72f9dfde-740c-4391-a75e-39e8535a83c3';
+    $pane->panel = 'alpha';
+    $pane->type = 'search_result';
+    $pane->subtype = 'search_result';
+    $pane->shown = TRUE;
+    $pane->access = array();
+    $pane->configuration = array(
+      'override_title' => 0,
+      'override_title_text' => '',
+      'override_title_heading' => 'h2',
+    );
+    $pane->cache = array();
+    $pane->style = array(
+      'settings' => NULL,
+    );
+    $pane->css = array();
+    $pane->extras = array();
+    $pane->position = 0;
+    $pane->locks = array();
+    $pane->uuid = '72f9dfde-740c-4391-a75e-39e8535a83c3';
+    $display->content['new-72f9dfde-740c-4391-a75e-39e8535a83c3'] = $pane;
+    $display->panels['alpha'][0] = 'new-72f9dfde-740c-4391-a75e-39e8535a83c3';
+  $display->hide_title = PANELS_TITLE_FIXED;
+  $display->title_pane = '0';
+  $handler->conf['display'] = $display;
+  $page->default_handlers[$handler->name] = $handler;
+  $pages['search_node'] = $page;
+
+  return $pages;
+
+}
```

Comment:
Fine! Even our in-house Drupal developer thought it might be nice with a comment, though.